core/nats-macros/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nats-macros"
```

Comment:
We should also add this to the workspace `Cargo.toml` at the root of this repo so that `cargo ws` (workspace management) is aware of it. This helps us keep versions of related crates in lockstep, among other things.

Reply:
uhh didnt i do that?

I added core/nats-macros to the Cargo.toml in the root.

do i have to do it somewhere else also for workspaces to work?

Reply:
I think just that should be enough for cargo ws.

Let's also rename it to `plane-core-nats-macro` so that it is under the `plane-` prefix when we push it to crates.io.